modules/seo-tools/jetpack-seo.php

```diff
@@ -114,30 +114,14 @@ public function meta_tags() {
 		 *
 		 * @param string Name of the site host. Defaults to empty string.
 		 */
-		$site_host     = apply_filters( 'jetpack_seo_site_host', '' );
-
-		$meta['title'] = sprintf( _x( '%1$s', 'Site Title', 'jetpack' ), get_bloginfo( 'title' ) );
-
-		if ( ! empty( $site_host ) ) {
-			$meta['title'] = sprintf(
-				_x( '%1$s on %2$s', 'Site Title on WordPress', 'jetpack' ),
-				get_bloginfo( 'title' ),
-				$site_host
-			);
-		}
+		$site_host = apply_filters( 'jetpack_seo_site_host', '' );
```

Comment:
Since we don't use `$site_host` anymore, maybe we should delete and deprecate this filter?
https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L5930